Samples/ResourceManagement/cpp-console-unpackaged/console_unpackaged_app.vcxproj

```diff
@@ -3,6 +3,7 @@
   <Import Project="packages\Microsoft.WindowsAppSDK.1.0.0-preview1\build\native\Microsoft.WindowsAppSDK.props" Condition="Exists('packages\Microsoft.WindowsAppSDK.1.0.0-preview1\build\native\Microsoft.WindowsAppSDK.props')" />
   <Import Project="packages\Microsoft.Windows.SDK.BuildTools.10.0.20348.19\build\Microsoft.Windows.SDK.BuildTools.props" Condition="Exists('packages\Microsoft.Windows.SDK.BuildTools.10.0.20348.19\build\Microsoft.Windows.SDK.BuildTools.props')" />
   <Import Project="packages\Microsoft.Windows.CppWinRT.2.0.210714.1\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('packages\Microsoft.Windows.CppWinRT.2.0.210714.1\build\native\Microsoft.Windows.CppWinRT.props')" />
+  <Import Project="..\..\Build.Windows.PlatformToolset.props" />
```

Comment:
I think this might need to be 
`  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />` to work in some cases.

(for all project files... sorry for the churn...)

Reply:
Hmmm I'm not sure I understood...

Do you mean that whenever we import Build.Windows.PlatformToolset.props, we should add the line you mentioned right before it?

Reply:
Ah, sorry for not being clear...

What I meant was that this:
`<Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />`
must come *before* this:
`<Import Project="..\..\Build.Windows.PlatformToolset.props" />`

However, doing some more investigating, I'm not sure if this is really needed or not though.
If the samples work fine, then it's probably not needed.